<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLogin</class>
 <widget class="QDialog" name="DialogLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>客户登陆界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
